src/symtab2gb/symtab2gb_main.cpp

```diff
@@ -0,0 +1,18 @@
+/******************************************************************\
+
+Module: symtab2gb_main
+
+Author: Diffblue Ltd.
+
+\******************************************************************/
+
+/// \file
+/// symtab2gb Main Module
+
+#include "symtab2gb_parse_options.h"
+
+int main(int argc, const char *argv[])
+{
+  symtab2gb_parse_optionst parse_options{argc, argv};
```

Comment:
Is there a reason for `{` `}` rather than `(` `)` ?

Reply:
Brace initialisation avoid parsing ambiguities and prevents implicit conversions, and is recommended "by default" by the [CPP Core Guidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es23-prefer-the--initializer-syntax). Unless we have a good reason my preference would be to stick to those as the 'default'.

Reply:
@martin-cs Preferred since #3546